src/main/java/com/springboot/star_wars_character_popularity/app/model/MovieModel.java

```diff
@@ -21,6 +21,10 @@
     @Column(name = "videoName")
     public String name;
 
+    public MovieModel(String name){
+        this.name = name;
+    }
```

Comment:
I guess it's better to use @Builder annotation on top of the class and to use it when you are initializing object, to prevent this boilerplate code.